lib/usd/utils/MergePrims.h

```diff
@@ -71,7 +43,6 @@ bool mergePrims(
     const PXR_NS::UsdStageRefPtr& dstStage,
     const PXR_NS::SdfLayerRefPtr& dstLayer,
     const PXR_NS::SdfPath&        dstPath,
-    bool                          mergeChildren = false,
-    MergeVerbosity                verbosity = MergeVerbosity::Default);
+    const MergePrimsOptions&      options);
```

Comment:
Looks like at least one call site is using default MergePrimOptions, so you could consider a default value here.

Reply:
I expect that to change shortly, as we add the UI to control the pull and push options, so it does not seems worthwhile.